ehrql/tables/beta/tpp.py

```diff
@@ -235,80 +270,130 @@ class clinical_events(EventFrame):
+        ),
     )
     sus_hrg_code = Series(
         str,
-        description="TODO",
+        description=(
+            "The core Healthcare Resource Group (HRG) code derived by sus+, "
+            "used for tariff application."
+        ),
     )
 
 
 @table
 class ec_cost(EventFrame):
+    """
+    Emergency care attendances data is provided via the NHS Secondary Uses Service.
+
+    This table gives details of attendance costs.
+    """
+
     ec_ident = Series(
         int,
         constraints=[Constraint.NotNull()],
-        description="TODO",
+        description="Unique identifier for the attendance used across the EC tables.",
     )
     grand_total_payment_mff = Series(
         float,
-        description="TODO",
+        description=(
+            "The grand total payment for the activity (`Net_SLA_Payment + Tariff_MFF_Payment`) "
+            "where SLA = service level agreement, "
+            "i.e. all contractual payments which is national tariff for the type of activity "
+            "**plus** any additional payments **minus** any applicable deductions. "
+            "MFF = Market Forces Factor, a geography-based cost adjustment)."
+        ),
     )
     tariff_total_payment = Series(
         float,
-        description="TODO",
+        description="The total payment according to the national tariff.",
     )
     arrival_date = Series(
         datetime.date,
-        description="TODO",
+        description=(
+            "The date the patient self presented at the accident & emergency department, "
+            "or arrived in an ambulance at the accident & emergency department."
+        ),
     )
     ec_decision_to_admit_date = Series(
         datetime.date,
-        description="TODO",
+        description="The date a decision to admit was made (if applicable).",
     )
     ec_injury_date = Series(
         datetime.date,
-        description="TODO",
+        description="The date the patient was injured (if applicable).",
     )
 
 
 @table
 class emergency_care_attendances(EventFrame):
-    id = Series(int)  # noqa: A003
-    arrival_date = Series(datetime.date)
-    discharge_destination = Series(SNOMEDCTCode)
+    """
+    Emergency care attendances data is provided via the NHS Secondary Uses Service.
+
+    This table gives details of attendances.
+
+    Note that there is a limited number of diagnoses allowed within this dataset,
+    and so will not match with the range of diagnoses allowed in other datasets
+    such as the primary care record.
+    """
+
+    id = Series(  # noqa: A003
+        int,
+        constraints=[Constraint.NotNull()],
+        description="Unique identifier for the attendance used across the EC tables.",
+    )
+    arrival_date = Series(
+        datetime.date,
+        description=(
+            "The date the patient self presented at the accident & emergency department, "
+            "or arrived in an ambulance at the accident & emergency department."
+        ),
```

Comment:
question: What's the preferred way of formatting multi-line strings? In this file, there's a mixture of parentheses, and triple quotes?